# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libgpod/gcc43.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- libgpod-0.6.0/tests/test-rebuild-db.cc.vanilla	2008-04-09 10:40:14.000000000 +0000
+++ libgpod-0.6.0/tests/test-rebuild-db.cc	2008-04-09 10:40:21.000000000 +0000
@@ -29,6 +29,7 @@
 #include <sys/stat.h>
 #include <typeinfo>
 #include <unistd.h>
+#include <string.h>
 
 #include <glib/gstdio.h>
 
